src/components/components/ebay-tooltip-overlay/index.marko

```diff
@@ -0,0 +1,81 @@
+import constants from "./constants"
+
+static var typeRoles = constants.typeRoles;
+
+static var pointerStyles = constants.pointerStyles;
+
+$ {
```

Comment:
Would be nice to merge these two scriptlets.
There is a few places with code like this (also for static blocks that could be merge).